src/pages/home/components/feature-description/feature-explanation-mobile.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { spaces, breakpoints, colors, textStyles } from '../../../../styles'
+import styled from 'styled-components'
+import Anchor from "../../../../components/anchor"
+
+const Headline = styled.h3`
+  margin: ${spaces.relax}px ${spaces.comfortable}px;
+
+  text-align: left;
+
+  ${textStyles.h3Bold}
+
+  @media screen and (min-width: ${breakpoints.desktop}px) {
+    margin: 0 
+    ${textStyles.h2Bold}
+  }
+`
+
+const Checklist = styled.img`
+  width: ${spaces.relax}px;
+  height: ${spaces.relax}px;
+  align-self: center;
+  margin-right: ${spaces.cozy}px;
+`
+
+const MyDiv = styled.div`
+  display: flex;
+  margin: 0 ${spaces.comfortable}px ${spaces.comfortable}px ${spaces.comfortable}px;
+`
+
+const Text = styled.div`
+${textStyles.h5}
```

Comment:
whitespace at the left

Reply:
huh? what do you mean?

Reply:
if you see all your css styles are being indented with a tab.